elements/pl-file-editor/pl-file-editor.mustache

```diff
@@ -42,6 +42,19 @@ $(function() {
                             <label for="modal-{{uuid}}-themes">Editor Theme</label>
                             <select class="custom-select" id="modal-{{uuid}}-themes">
                             </select>
+
+                            <div id="modal-split" style="padding-top: 1rem"></div>
```

Comment:
Instead of a spacer div, you should wrap your new controls in a div with the `form-group` class. See https://getbootstrap.com/docs/4.6/components/forms/ for an example.